upgrade/breaking-changes/3-0-0.md

```diff
@@ -0,0 +1,44 @@
+---
+title: 3.0.0
+description: Handle the changes in the 3.0.0 release of the Telerik UI for Blazor components.
+page_title: Breaking Changes in 3.0.0
+slug: changes-in-3-0-0
+position: 1
+---
+
+# Upcoming Breaking Changes in 3.0.0
+
+> The 3.0.0 release will be shipped in mid January. The following list of breaking changes will be updated with detailed explanation how to migrate your project.
+
+## Component Changes
+
+- TileLayout - Introduce identifier of the TileLayout item. OnResize and OnReorder events will receive EventArgs that will point to the item that has been resized/reordered. This items comes from our [user voice](https://feedback.telerik.com/blazor/1489011)
+- TreeView - add `ExpandedItems` parameter to remove `ExpandedField`
+- Grid – change `OnRead` method to return the `Data`
```

Comment:
This sounds like a major change. Is there anything else we can mention, even on high-level?

```suggestion
- Grid – change `OnRead` method to return the `Data`
```